certified-connectors/Strategic Portfolio Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7505 @@
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "Values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Item"
+          }
+        }
+      }
+    },
+    "EntityMilestone": {
+      "type": "object",
+      "properties": {
+        "MilestoneId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "ActualDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "Description": {
+          "type": "string"
+        },
+        "Name": {
+          "type": "string"
+        },
+        "PlannedDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "Status": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "TransitionalActualDate": {
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "RemoteFinancialValuesEventArgs": {
+      "type": "object",
+      "properties": {
+        "SubscriptionToken": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true,
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "EntityUid": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true,
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "EnterpriseFinancialTypeUid": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true,
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "DimensionUids": {
+          "type": "array",
+          "items": {
+            "format": "uuid",
+            "type": "string",
+            "example": "00000000-0000-0000-0000-000000000000"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "RemoteEntityEventArgs": {
+      "type": "object",
+      "properties": {
+        "SubscriptionToken": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true,
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "EntityUid": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true,
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "EntityName": {
+          "type": "string",
+          "readOnly": true
+        },
+        "EntityFields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EntityField"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "EntityField": {
+      "type": "object",
+      "properties": {
+        "Uid": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "Name": {
+          "type": "string"
+        },
+        "Value": {}
```

Comment:
@sanda-cheudan can you please explain this empty Value? can you please define or remove?